frontend/src/index.tsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import {BrowserRouter} from 'react-router-dom'
 
 ReactDOM.render(
-    <React.StrictMode>
```

Comment:
Warum hast du den strict mode entfernt? Damit verlierst du evtl. Kompatibilität zu neuen React Versionen.

Reply:
Gewöhnungssache. Hatte ich bei meinem letzten Capstone genauso. Aber ich habe den Browserrouter in die App.tsx gewechselt